src/components/modals/Modal.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import ReactModal from 'react-modal';
+import PropTypes from 'prop-types';
+import cn from 'classnames';
+
+import ModalHeader from './ModalHeader';
+
+import './Modal.css';
+
+ReactModal.setAppElement('#root');
+
+class Modal extends Component {
+  static Theme = {
+    default: 'default',
+    warning: 'warning',
+    danger: 'danger',
+    info: 'info',
+  };
+
+  static WidthMode = {
+    default: 'default',
+    wide: 'wide',
+    narrow: 'narrow',
+  };
+
+  static HeightMode = {
+    top: 'top',
+    center: 'center',
+    fullFill: 'full-fill',
+  };
+
+  static propTypes = {
+    theme: PropTypes.string,
+    widthMode: PropTypes.string,
+    header: PropTypes.string,
+  };
+
+  static defaultProps = {
+    theme: Modal.Theme.default,
+    widthMode: Modal.WidthMode.default,
+    heightMode: Modal.HeightMode.center,
+  };
+
+  render() {
+    const { children, theme, widthMode, heightMode, header, onRequestClose, ...rest } = this.props;
+
+    return (
+      <ReactModal
+        className={cn({
+          [`ReactModal__Content--theme-${theme}`]: theme,
+          [`ReactModal__Content--width-mode-${widthMode}`]: widthMode,
+        })}
+        overlayClassName={cn('ReactModal__Overlay-default', {
+          [`ReactModal__Overlay--height-mode-${heightMode}`]: heightMode,
+        })}
+        shouldCloseOnOverlayClick={true}
```

Comment:
you do not have to pass true for boolean options.

```
<ReactModal
  {/* ...props */}
  shouldCloseOnOverlayClick
  shouldCloseOnEsc
/>
```

But if linter forces to do so, then it's okay :)

Reply:
Yeah, I know, but I prefer clearly indicate, in a case when I pass only `true` value :)